exporter/collector/integrationtest/testdata/fixtures/ops_agent_host_metrics_expect.json

```diff
@@ -3150,6 +3150,486 @@
+          },
+          {
+            "key": "process"
+          }
+        ],
+        "metricKind": "CUMULATIVE",
+        "valueType": "INT64",
+        "unit": "s",
+        "description": "Total CPU seconds broken down by different states.",
+        "displayName": "processes/cpu_time"
+      }
+    },
+    {
+      "metricDescriptor": {
+        "type": "agent.googleapis.com/processes/rss_usage",
+        "labels": [
+          {
+            "key": "command"
+          },
+          {
+            "key": "command_line"
+          },
+          {
+            "key": "owner"
+          },
+          {
+            "key": "pid"
+          },
+          {
+            "key": "process"
+          }
+        ],
+        "metricKind": "GAUGE",
+        "valueType": "DOUBLE",
+        "unit": "By",
+        "description": "The amount of physical memory in use.",
+        "displayName": "processes/rss_usage"
+      }
+    },
+    {
+      "metricDescriptor": {
+        "type": "agent.googleapis.com/processes/vm_usage",
+        "labels": [
+          {
+            "key": "command"
+          },
+          {
+            "key": "command_line"
+          },
+          {
+            "key": "owner"
+          },
+          {
+            "key": "pid"
+          },
+          {
+            "key": "process"
+          }
+        ],
+        "metricKind": "GAUGE",
+        "valueType": "DOUBLE",
+        "unit": "By",
+        "description": "Virtual memory size.",
+        "displayName": "processes/vm_usage"
+      }
+    },
+    {
+      "metricDescriptor": {
+        "type": "agent.googleapis.com/processes/disk/read_bytes_count",
+        "labels": [
+          {
+            "key": "command"
+          },
+          {
+            "key": "command_line"
+          },
+          {
+            "key": "owner"
+          },
+          {
+            "key": "pid"
+          },
+          {
+            "key": "process"
+          }
+        ],
+        "metricKind": "CUMULATIVE",
+        "valueType": "INT64",
+        "unit": "By",
+        "description": "Disk bytes transferred.",
+        "displayName": "processes/disk/read_bytes_count"
+      }
+    },
+    {
+      "metricDescriptor": {
+        "type": "agent.googleapis.com/processes/disk/write_bytes_count",
+        "labels": [
+          {
+            "key": "command"
+          },
+          {
+            "key": "command_line"
+          },
+          {
+            "key": "owner"
+          },
+          {
+            "key": "pid"
+          },
+          {
+            "key": "process"
+          }
+        ],
+        "metricKind": "CUMULATIVE",
+        "valueType": "INT64",
+        "unit": "By",
+        "description": "Disk bytes transferred.",
+        "displayName": "processes/disk/write_bytes_count"
+      }
+    },
+    {
+      "metricDescriptor": {
+        "type": "agent.googleapis.com/swap/percent_used",
+        "metricKind": "GAUGE",
+        "unit": "By",
```

Comment:
FYI - this unit is wrong.  It's also wrong in the underlying data.  We may want to re-capture ops agent host metrics and see if we can pull the version where this unit is fixed before we turn on CMD for ops-agent.